public/templates/header.php

```diff
@@ -26,6 +26,7 @@
     <h1>COMP 5531 Database App</h1>
     <ul class="topnav">
       <li class="navelem"><a class="active" href="index.php">Home</a></li>
+      <li class="navelem"><a  href="notices.php">Announcements</a></li>
       <li class="dropdown">
         <button class="dropbtn">Courses</button>
```

Comment:
btw is there a reason why the model is called `notice` but we display them as `announcements`?

Reply:
because I realized once everything was implemented that announcements is a better name for it, but notice is shorter, so I decided to keep it

Reply:
Okay then let's rename all of the notices to announcements?

Reply:
done